config/locales/client.en.yml

```diff
@@ -1038,6 +1038,7 @@ en:
     user_fields:
       none: "(select an option)"
       required: 'Please enter a value for "%{name}"'
+      same_as_password: 'Your password is the same.'
```

Comment:
How about something that explains the user why this is considered an error?

```suggestion
      same_as_password: 'For security reasons, your password should not be repeated in other fields.'
```

Reply:
I believe that is too long and does not look very good:

<img width="587" alt="image" src="https://user-images.githubusercontent.com/23153890/194909191-3bc2308e-dbc2-4375-9bf9-1c42c6804104.png">